src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Random randomNumber = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+
+        Ball randomBall = new Ball(colorSupplier.getRandomColor(), randomNumber.nextInt(100));
```

Comment:
you should declare a constant for the magic number `100` :)

something like `MAX_BALL_NUMBER`

Reply:
![image](https://github.com/mate-academy/jv-lottery/assets/141054191/14abd425-6acf-4713-819e-dbd80fe601de)
don`t know how to handle it

Reply:
static keyword helped